VirtoCommerce.Storefront.Model/Cart/Demo/ConfiguredGroup.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using VirtoCommerce.Storefront.Model.Catalog;
+using VirtoCommerce.Storefront.Model.Common;
+
+namespace VirtoCommerce.Storefront.Model.Cart.Demo
+{
+    public class ConfiguredGroup : Entity
+    {
+        public ConfiguredGroup(int quantity, Currency currency, string productId)
+        {
+            Id = Guid.NewGuid().ToString("N");
+            ProductId = productId;
+            Quantity = quantity;
+            Currency = currency;
+
+            ExtendedPrice = new Money(currency);
+            ExtendedPriceWithTax = new Money(currency);
+            TaxTotal = new Money(currency);
+
+            ListPrice = new Money(currency);
+            ListPriceWithTax = new Money(currency);
+            SalePrice = new Money(currency);
+            SalePriceWithTax = new Money(currency);
+            PlacedPrice = new Money(currency);
+            PlacedPriceWithTax = new Money(currency);
+        }
+
+        public string ProductId { get; set; }
+
+        public Product Product { get; set; }
+
+        public DateTime CreatedDate { get; set; }
+
+        public DateTime? ModifiedDate { get; set; }
+
+        public string CreatedBy { get; set; }
+
+        public string ModifiedBy { get; set; }
+
+        [JsonRequired]
+        public virtual IList<LineItem> Items { get; set; } = new List<LineItem>();
```

Comment:
It is better to initialize all properties in the one place.
Please move collection initialization to constructor.